ipatests/test_webui/ui_driver.py

```diff
@@ -769,7 +769,8 @@ def fill_text(self, selector, value, parent=None):
             parent = self.get_form()
         tb = self.find(selector, By.CSS_SELECTOR, parent, strict=True)
         try:
-            tb.clear()
+            tb.send_keys(Keys.CONTROL + 'a')
+            tb.send_keys(Keys.DELETE)
```

Comment:
Not sure if it might break other tests. It's probably fine. Might be worth to run not just range test suite in the temp commit.

Reply:
Agree. I've added all WebUI tests to PR CI config, so we will see if something has broken.

Reply:
@pvoborni 
No regressions found.